app/assets/stylesheets/form.scss

```diff
@@ -0,0 +1,4 @@
+.error {
+  border-color: #cc4c3f;
```

Comment:
Color literals like `#cc4c3f` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, border-color